src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -67,4 +72,20 @@ object TestFormulas {
       _ <- validateNoCacheHits(metadata, workflow.testName)
     } yield ()
   }
+
+  def runFinalDirsWorkflow(wf: Workflow, dirOption: String): Test[Unit] =  {
+    import spray.json.DefaultJsonProtocol._
+
+    def dirSize = {
+      val options = wf.data.options.get
+      val outputDir = parse(options).toOption.flatMap(_.\\(dirOption).head.asString).get
```

Comment:
ToL: Incidentally, that `.\\` looks like it's crying out to be a DSL-y inlined function